lib/govuk_index/presenters/indexable_content_presenter.rb

```diff
@@ -0,0 +1,64 @@
+module GovukIndex
+  class IndexableContentPresenter
+    DEFAULTS = %w(body parts).freeze
+    BY_FORMAT = {
+      'contact'     => %w(title description),
+      'licence'     => %w(licence_short_description licence_overview),
+      'transaction' => %w(introductory_paragraph more_information),
+    }.freeze
+
+    def initialize(format:, details:, sanitiser:)
+      @format    = format
+      @details   = details
+      @sanitiser = sanitiser
+    end
+
+    def indexable_content
+      return nil if details.nil?
+      sanitiser.clean(indexable)
+    end
+
+  private
+
+    attr_reader :details, :format, :sanitiser
+
+    def indexable
+      case format
+      when 'contact'
+        contact
+      else
+        default_indexable_content
```

Comment:
why do we need the conditional? if we do need the conditional shouldn't it be extended to cover formats that can have hidden content?

could we not just make this 

```indexable_content_parts + hidden_content + content_group_titles```

that would keep the existing pattern.